### PR TITLE
LPS-140223 It is possible to leave a layout with a Relationship tab on first place by removing the fields tab

### PR DESCRIPTION
**Pull request by daiazius**
Related issue: https://issues.liferay.com/browse/LPS-140223

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/index.tsx [hunk 1]
```diff
@@ -200,6 +200,15 @@ const Layout: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 			return;
 		}
 
+		if (objectLayout.objectLayoutTabs[0].objectRelationshipId > 0) {
+			openToast({
+				message: Liferay.Language.get('an-error-occurred'),
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/index.tsx:205]: Hey @daiazius, as you are adding validation, you can change this error message to be more specific, talk to the object design team, Victor Santos or Julia Lira, they will tell you the best message to put here.

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @daiazius, I made a suggestion, let me know if you have any questions

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**